compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -243,6 +243,12 @@ def AMDAIEFlattenLogicalObjectFifo :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEFlattenLogicalObjectFifoPass()";
 }
 
+def AMDAIELinalgFunctionOutlining :
+  Pass<"iree-amdaie-linalg-function-outlining", "ModuleOp"> {
+  let summary = "Outlining of linalg compute ops";
```

Comment:
Can you add some additional information in a description = [{ }] block about which linalg ops specifically are outlined (i.e. linalg.fill is not) and what the motivation for this pass is, and what some assumptions are? Sorry, I know that most of our passes in this file don't contain descriptions, but I think it'd be a good thing if we all started adding high-level information here for people without context.

Reply:
I have added that. Please check if the wordings are correct.

Reply:
Thanks. 

Is 

```
One assumption this pass currently makes: All elementwise/matmul linalg ops within a
    dispatch have same body content.
```

still true? Otherwise looks good.

Reply:
Same question. Doesnt operation equivalence check the region as well?

Reply:
I forgot to remove that from the `.td` after adding the Equivalence check. I've removed it now. Operation equivalence indeed checks the region as well.